Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -162,15 +164,17 @@ fun ChannelNameBar(
                     modifier = Modifier
                         .clickable(onClick = {}) // TODO: Show not implemented dialog.
                         .padding(horizontal = 12.dp, vertical = 16.dp)
-                        .preferredHeight(24.dp)
+                        .preferredHeight(24.dp),
+                    contentDescription = null
```

Comment:
Shouldn't this two icons (search and info) contain `contentDescription`? They don't seem to be decorative icons